src/inventories/quantities/model/AdjustmentBySku.ts

```diff
@@ -1,9 +1,11 @@
+import { AdjustmentType } from '../../enum/AdjustmentType';
 import { InventoryStatus } from '../../enum/InventoryStatus';
 
 export default interface AdjustmentBySku {
   quantity: number;
   inventoryStatus: InventoryStatus;
   deduplicationKey: string;
   locationId: string;
+  operation: AdjustmentType;
```

Comment:
Why did you have to add this here? The list in the metadata should specify whether or not it's a set or an adjusts, right?

Reply:
@ryankazokas I can change the mapping in the playbook to do it that way. I thought this approach simplified the implementation of the mapping. Let me if this should change to not included

Reply:
I'm a little confused here. We are using two lists right?
inventorySets
inventoryAdjusts

which are a list with AdjustmentBySku? I'm just saying it's a little redudant and little risky to have the operation embedded within the adjustment list like that. What if i put an adjust operation inside an adjustment in the inventorySets list, which is honored? SETS or ADJUSTS?

Reply:
@ryankazokas Actually I'm trying to create this request According to the baches API docs.


{
    "businessId": "938a69a8-9ef2-4faa-b485-dad486aba56e",
    "adjustments": [
        {
            "idempotentKey": "123456789",
            "inventoryItemId": "703",
            "quantity": 10,
            "operation": "ADJUST",
            "locationId": 51,
            "inventoryStatus": "ON_ORDER",
            "memo": "Reason the adjustment was made."
        },
        {
            "idempotentKey": "123456789",
            "inventoryItemId": "703",
            "quantity": 12,
            "operation": "SET",
            "locationId": 51,
            "inventoryStatus": "COMMITTED",
            "memo": "Reason the adjustment was made."
        }
    ]
}

Reply:
Yea, i understand it's needed to actually make the call to the api. I'm just trying to make this transition seamless though. Would it be easier to have two interfaces(AdjustmentBySku, AdjustmentBySkuWithOperation which extends the former with just oepration) so that when you are setting it in the inventory steps it maps it the way we've done in the past, then when we are actually ready to make the API call if it's going to use the batch inventory it remaps it with the operation accordingly based on which list

Step 1 - create inventory adjustment lists for inventorySets/adjusts no operation
Step 2 - Update channelape, go through list of inventorySets, and inventoryAdjusts list and add the operation
Step 3 - Make API call to the batch inventory api